image/templates/helm/stackrox-secured-cluster/internal/config-shape.yaml

```diff
@@ -129,6 +129,10 @@ collector:
   priorityClassName: null # string
   disableSELinuxOptions: null # bool
   seLinuxOptionsType: null # string
+  containerd:
+    enabled: true # bool
```

Comment:
This is not how this is intended to be used.
The config-shape mechanism is not for defaults, it is really just there for making sure that the configuration tree can be safely accessed without checking for existence of sub-dicts everywhere along the way.
You should not see any non-null settings in the config-shape, more precisely: either dicts or null-keys.

Reply:
I didn't know about this or the `internal/defaults` file, I'll get it sorted, thanks @mclasmeier and @ludydoo for the comments!